packages/expo/src/Expo.ts

```diff
@@ -53,6 +52,8 @@ export { FileSystem } from 'expo-file-system';
 export { Font };
 export { GLView } from 'expo-gl';
 export { GoogleSignIn } from 'expo-google-sign-in';
+import * as Haptics from 'expo-haptics';
+export { Haptics };
```

Comment:
Can we warn people if they still use `Haptic` instead of `Haptics`? Take a look at `Object.defineProperties` at the bottom of the file, you can define a custom getter there.